src/components/FeatureStore/featureStore.util.js

```diff
@@ -345,14 +352,18 @@ export const handleFetchData = async (
 
     if (result) {
       data.content = parseFeatureSets(result)
-      data.yamlContent = result
+      data.originalContent = result
     }
   } else if (pageTab === FEATURES_TAB) {
-    result = await fetchFeatures(project, filters)
+    const allResult = await Promise.all([
+      fetchFeatures(project, filters),
+      fetchEntities(project, filters)
+    ])
+    result = flatten(allResult)
```

Comment:
Since ES2019/ES10 the standard has an `Array.prototype.flat` method:
```suggestion
    result = allResult.flat()
```